app/src/main/java/com/android/sample/ui/library/Library.kt

```diff
@@ -188,3 +144,54 @@ fun TitleWithArrow(title: String, onClick: () -> Unit) {
                 })
       }
 }
+
+@Composable
+fun PageTitle(mainTitle: String, mainTitleTag: String) {
+  Box(
+      modifier =
+          Modifier.fillMaxWidth()
+              .drawWithCache {
+                // Apply the bottom border or shadow in dark mode
+                onDrawWithContent {
+                  drawContent()
+                  drawLine(
+                      color = BorderColor,
+                      strokeWidth = 1.dp.toPx(),
+                      start = Offset(0f, size.height), // Bottom left
+                      end = Offset(size.width, size.height) // Bottom right
+                      )
+                }
+              }
+              .shadow(elevation = 2.dp, spotColor = ShadowColor, ambientColor = ShadowColor)
+              .height(48.dp)
+              .background(color = MaterialTheme.colorScheme.background)) {
+        Text(
+            text = mainTitle,
+            color = MaterialTheme.colorScheme.primary,
+            style = MaterialTheme.typography.headlineLarge,
+            modifier = Modifier.padding(top = 14.dp).testTag(mainTitleTag))
+      }
+}
+
+@Composable
+fun CornerIcons(
+    onClick: () -> Unit,
+    icon: ImageVector,
+    contentDescription: String,
+    modifier: Modifier = Modifier,
+    iconSize: Dp = 28.dp,
+    gradientBrush: Brush = IconsGradientBrush
+) {
+  IconButton(onClick = onClick, modifier = modifier) {
+    Icon(
+        imageVector = icon,
+        contentDescription = contentDescription,
+        modifier =
+            Modifier.size(iconSize).graphicsLayer(alpha = 0.99f).drawWithCache {
+              onDrawWithContent {
+                drawContent()
+                drawRect(gradientBrush, blendMode = BlendMode.SrcAtop)
+              }
+            })
+  }
+}
```

Comment:
I've noticed that this composable is also used in other packages, like profile and authentication. I would suggest moving it to a separate shared UI package (e.g., ui.components) to improve readability.